src/libasr/pass/global_stmts_program.cpp

```diff
@@ -19,7 +19,7 @@ using LFortran::ASRUtils::EXPR;
  */
 void pass_wrap_global_stmts_into_program(Allocator &al,
             ASR::TranslationUnit_t &unit) {
-    std::string fn_name_s = "_lfortran_main_program";
+    std::string fn_name_s = "_lpython_main_program";
```

Comment:
I see. That's not going to work, as it will break LFortran. What we need is to pass the name as an argument to this function, then LPython can set it to `_lpython_main_program`, and LFortran can set it to `_lfortran_main_program`.

Reply:
Everything in `libasr` is shared with LFortran, so it needs to be language neutral.